### DISCUSSION TITLE
differences in code syntax

### DISCUSSION THREAD

**Discussion by pierre-jeanne** [idaholab/moose#22824] [Q&A General] [answered]
Hello,
i am going through the tutorial and I was wondering what are the differences between, for example:

[Variables]
  [./porepressure]
  [../]
[]

and
[Variables]
  [porepressure]
  []
[]

And the difference between:

[Variables]
  [porepressure]
  initial_condition = 2
  []
[]

and 

[Variables]
  [porepressure]
  []
[]
[Variables]
  [porepressure]
  []
[]
[ICs]
  [pressure_ic]
    type = ConstantIC
    variable = porepressure
    value = 2
  []
[]

Are they the same?
P.

**Comment by hugary1995** [comment, marked as answer]
1. Yes. `[] []` is preferred over `[./] [../]`
2. Yes. `initial_condition = 2` is a shortcut to what's shown below.
